harness/src/main/java/org/apache/geode/perftest/TestConfig.java

```diff
@@ -161,24 +154,4 @@ public void jvmArgs(String role, String... jvmArgs) {
     return Collections.unmodifiableMap(jvmArgs);
   }
 
-  public static class TestStep {
```

Comment:
made this a top-level class because I added `equals()` and `hashCode()` and needed a test for it